locations/segen.yml

```diff
@@ -128,7 +129,9 @@ networks:
     ipv6_subprefix: 1
     assignments:
       segen-core: 1
-      segen-switch: 2
+      segen-switch-1: 2
+      segen-switch-2: 3
+      segen-switch-octagon: 3
```

Comment:
These both have the same assignment.  If that works as expected (I have no idea), then maybe it's good to add a comment so it's understood in the future.